src/components/Navbar.css

```diff
@@ -26,18 +26,22 @@
 }
 
 .navbar-logo {
-  -webkit-text-stroke: 0.5px black;
+  display: block;
   color: #fff;
   justify-self: start;
   margin-left: 10px;
   margin-top: 93px;
   cursor: pointer;
   text-decoration: none;
   font-size: 9rem;
-  display: flex;
-  align-items: flex-end;
-  font-family:'Anton';
-  font-display: optional;
+  height: 126px;
+  aspect-ratio: 412/126;
+  background: url('../assets/images/logo.svg') no-repeat;
+  background-size: contain;
+  background-position: top left;
+  text-indent: 100%;
+  white-space: nowrap;
+  overflow: hidden;
```

Comment:
Hides actual text-content (in DOM) by
* pushing text out of its container (`text-indent: 100%`)
* disabling line-wrapping (`white-space: nowrap`)
* and hiding overflow (`overflow: hidden`)

Reply:
I don't quite understand what this is doing to the logo but will have a play around with it to try and understand it a bit more.